Airgap install (#1657)
spectrocloud/librarium: docs/docs-content/vertex/install-palette-vertex/airgap/vmware-vsphere-airgap-instructions.md

---
sidebar_label: "VMware vSphere Airgap Instructions"
title: "VMware vSphere Airgap Instructions"

> description: "Learn how to install VerteX in an air gap environment."

Author: ```suggestion
description: "Learn how to install Palette in an air gap environment."
```

  <br />

    - `ECR_IMAGE_REGISTRY`: The IP address or domain name of the public OCI registry for images.
    - `ECR_IMAGE_BASE`: The namespace or repository name that hosts the VerteX images.
    - `ECR_IMAGE_REGISTRY_REGION`: The AWS region where the ECR registry is located.
    - `ECR_PACK_BASE`: The namespace or repository name that hosts the VerteX Packs.
    - `ECR_PACK_REGISTRY`: The IP address or domain name of the OCI registry.
    - `ECR_PACK_REGISTRY_REGION`: The AWS region where the ECR registry is located.

    ```shell
    export ECR_IMAGE_REGISTRY=<ecr-endpoint>
    export ECR_IMAGE_BASE=spectro-images
    export ECR_IMAGE_REGISTRY_REGION=us-east-1
    export ECR_PACK_REGISTRY=<ecr-endpoint>
    export ECR_PACK_BASE=spectro-packs
    export ECR_PACK_REGISTRY_REGION=us-east-1
    ```

    Example

    ```shell hideClipboard
    export ECR_IMAGE_REGISTRY=public.ecr.aws/1234567890
    export ECR_IMAGE_BASE=spectro-images
    export ECR_IMAGE_REGISTRY_REGION=us-east-1
    export ECR_PACK_REGISTRY=123456789.dkr.ecr.us-east-1.amazonaws.com
    export ECR_PACK_BASE=spectro-packs
    export ECR_PACK_REGISTRY_REGION=us-east-1
    ```

  </TabItem>
  </Tabs>

---

> 11. Download the airgap setup binary. Our support team will provide you with the proper version and credentials. Replace the values in the commands below with the recommended version and credentials provided by our support team.

Author: ```suggestion
11. Download the airgap setup binary. Replace the placeholder values in the commands below with the recommended version and credentials that our support team provides.
```

  ![Example of a file server hosting the unzipped manifest content](/enterprise-version_airgap_airgap-instructions_file-server-caddy.png)

3. Ensure your file server is accessible from the environment in which you are installing VerteX. Use the following command to verify the manifest content is accessible from the file server. Replace the hostname or IP address below with your file server hostname or IP address.

  ```shell
  curl http://<hostname>:<port>/roar/nickfury/versions.yaml
  ```

  ```yaml hideClipboard
    versions:
    - version: "3.3"
      filepath: "/roar/nickfury/3.3/version.yaml"
      patchVersionsFilepath: "/roar/nickfury/3.3/versions.yaml"
    - version: "3.4"
      filepath: "/roar/nickfury/3.4/version.yaml"
      patchVersionsFilepath: "/roar/nickfury/3.4/versions.yaml"
    - version: "4.0"
      filepath: "/roar/nickfury/4.0/version.yaml"
      patchVersionsFilepath: "/roar/nickfury/4.0/versions.yaml"
  ```

## Next Steps

> You are now ready to deploy the airgap VerteX installation. The important difference is that you will specify your OCI registry and file server during the installation process. Refer to the [VMware Install Instructions](../install-on-vmware/install-on-vmware.md) guide for detailed guidance on installing VerteX. 

Author: ```suggestion
You are now ready to deploy the airgap VerteX installation. Yoy will specify your OCI registry and file server during the installation process. Refer to the [VMware Install Instructions](../install-on-vmware/install-on-vmware.md) guide for detailed guidance on installing VerteX. 
```
